packages/components/notification/src/notification.vue

```diff
@@ -84,7 +84,7 @@ const verticalProperty = computed(() =>
 const positionStyle = computed<CSSProperties>(() => {
   return {
     [verticalProperty.value]: `${props.offset}px`,
-    zIndex: currentZIndex.value,
+    zIndex: props.zIndex ?? currentZIndex.value,
```

Comment:
The default value of `zIndex` attribute  is `0`, so the `props.zIndex ?? currentZIndex.value` is always is `0`.

The default value of `zIndex` attribute is unreasonable, maybe we should remove it.